Examples/Framework/include/ActsExamples/EventData/SpacePointContainer.hpp

```diff
@@ -0,0 +1,125 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2024 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/EventData/SpacePointContainer.hpp"
+#include "Acts/Utilities/Holders.hpp"
+#include "ActsExamples/EventData/SimSpacePoint.hpp"
+
+#include <any>
+
+namespace ActsExamples {
+
+template <typename collection_t>
+class SpacePointContainer {
+ public:
+  using CollectionType = collection_t;
+  using ValueType = typename CollectionType::value_type;
```

Comment:
What is the benefit of templating the collection? I wonder if this could be a generic vector implementation instead and live in Core

Reply:
We may potentially need to pass a `DataVector` or any kind of collection the user may be using. I don't want to corner myself on `std::vector`s